dbms/src/Server/Server.cpp

```diff
@@ -1215,6 +1215,13 @@ int Server::main(const std::vector<std::string> & /*args*/)
     /// Init TiFlash metrics.
     global_context->initializeTiFlashMetrics();
 
+    /// Initialize users config reloader.
+    auto users_config_reloader = UserConfig::parseSettings(config(), config_path, global_context, log);
+
+    /// Load global settings from default_profile and system_profile.
+    global_context->setDefaultProfiles(config());
+    Settings & settings = global_context->getSettingsRef();
+
```

Comment:
Call `global_context->getBackgroundPool()` and sent it to `initializeRateLimiter` explicitly?
And add more comments about the dependencies between config settings and init background pool

Reply:
Done.